app/models/task.rb

```diff
@@ -7,21 +7,22 @@ class Task < ActiveRecord::Base
   belongs_to :company
   belongs_to :project
   belongs_to :assignable, polymorphic: true
-  validates :complexity
```

Comment:
you removed it! is it being fixed by tehreem?

Reply:
asked her. she said to remove it until her branch merges at the master